### ISSUE TITLE
How can I use multiple GPUs?

### ISSUE THREAD

**Issue by jneto04** [zalandoresearch/flair#389]
I have two GPUs (Tesla K80), how can I use FLAIR in parallel on GPUs?

Set:

Ubuntu 16.04
2 NVIDIA Tesla K80
10 vCPUs
40 GB RAM

**Comment by alanakbik** [comment]
Hi @jneto04 we don't yet support multi-GPU setups but have this as a TODO in the pipeline (together with FP16 support). Any help from the community will be greatly appreciated :)

**Comment by tabergma** [comment]
see also https://github.com/zalandoresearch/flair/issues/37

**Comment by tabergma** [comment]
As this issue is a duplicate of #37, we will close this issue. If you have any suggestions or questions regardings this topic, please refer to issue #37. Thanks.

**Comment by aditya-malte** [comment]
Hi @alanakbik,
Any updates on Multi-GPU support (you mentioned it was expected in v0.3)? Seems even more critical now as models have become a whole lot larger.
Thanks

**Comment by mallapraveen** [comment]
@ is this supported now? multi gpu training?
